### PR TITLE
fix: remove de.linkedin.com link

### PR DESCRIPTION
**Pull request by zepatrik**
diff should be self-explanatory

Or is there a web link that should be here instead?

### CODE REVIEW

**Comment by vinckr** [comment]
I have to ask pascal @pascal808, but I think there should be 
`https://think-speak-transform.com/`

**Comment by zepatrik** [comment]
Then check with him and merge once he approved

File: src/page-content/shared/content-team.tsx [hunk 1]
```diff
@@ -287,11 +287,11 @@ export const collaborators: TeamProps = {
       social: [
         {
           network: SocialNetworks.linkedin,
-          href: 'https://de.linkedin.com/in/pascalmorgan'
+          href: 'https://www.linkedin.com/in/pascalmorgan'
         },
         {
           network: SocialNetworks.web,
-          href: 'https://www.linkedin.com/in/pascalmorgan'
+          href: ' https://think-speak-transform.com/'
```
Comment (vinckr) [inline src/page-content/shared/content-team.tsx:294]:
```suggestion
          href: 'https://think-speak-transform.com/'
```
space

**Comment by vinckr** [comment]
I will confirm with Pascal at 1230, but should be ok to merge.

**Review by aeneasr — APPROVED** [review]

**Comment by Pascal808** [comment]
ack. thx, guys!

**Pull request merged**